broadcast/build.rs

```diff
@@ -0,0 +1,17 @@
+use std::io::Result;
+fn main() -> Result<()> {
+    // Proto compilation rules for `linked` dialect
+    let mut config = prost_build::Config::new();
+    config.bytes([
+        "Car.sequencer",
```

Comment:
If we decide to use the terms from Autobahn, I think we should call this `broadcast::autobahn`. 

Given Autobahn is a little more specific about how this should work (and includes consensus), I think we should instead use our own terminology and link to Autobahn in the root `mod.rs` as an inspiration.